### ISSUE TITLE
some errors

### ISSUE THREAD

**Issue by baba-hashimoto** [GHeinzelmann/BAT.py#2]
Thank you for making a very interesting program. I immediately ran a follow-up test of the program. However, there was a problem that "l00 to l08" and "t00 to t08" did not work when FE calculations were performed. If possible, please post the results of your calculations using the sample files on github.

**Comment by baba-hashimoto** [comment]
The reason the program didn't work was because there was a space at the end of the first line of the "disang.rest" file.

**Comment by GHeinzelmann** [comment]
I had not run into that particular problem, for me it always works normally. Did you have to edit the code to change the way the disang file is written? If so let me know, it might have to do with the AMBER version or the way your computer writes the files. I can try to fix it so it works for everybody. Thanks for bringing up the issue.

**Comment by GHeinzelmann** [comment]
I had not run into that particular problem, for me it always works normally. Did you have to edit the code to change the way the disang file is written? If so let me know, it might have to do with the AMBER version or the way your computer writes the files. I can try to fix it so it works for everybody. Thanks for bringing up the issue.

**Comment by baba-hashimoto** [comment]
I manually erased the last space at the end of the first line of the "disang.rest" file. I have not yet implemented any changes to the source. Unfortunately, I haven't calculated with the latest version of AMBER (AMBER18). I used the latest version of AmberTools when I used BAT.py. In my computing environment, I can't perform all the calculations fast enough. However, I think it is a useful program and I am using AMBER14 to implement the program. The error content is Segmentatin faults, so it may be an error in AMBER14's pmemd.cuda.
